gensrc/thrift/FrontendService.thrift

```diff
@@ -324,6 +324,14 @@ struct TTableStatus {
     5: optional i64 last_check_time
     6: optional i64 create_time
     20: optional string ddl_sql
+    // id for materialized view
+    21: optional string id
```

Comment:
why we need id for  materialized view? for table and view, we only have name, not id.